### PR TITLE
Edmond OK

### LINKED ISSUE #74: otuyishime Github Workflow
## Workflow steps
- [ ] Create a new branch
- [ ] Add commits
- [ ] Open a pull request
- [ ] Discuss and review your code
- [ ] Merge the code

### PR DESCRIPTION
**Pull request by olaboutcode**
I think this doc can help people find things to do in Edmond, OK.

closes #74 

fixes #74 

resolves #74

### CODE REVIEW

**Comment by raja777m** [comment]
it helps.!

File: OK.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Edmond, OK
+## Places to go
```
Comment (raja777m) [inline OK.md:3]: Nice places for outdoor activities in North Carolina.
:horse_racing: 
@Otuyishime

File: OK.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+### What to do
```
Comment (kdmarko) [inline OK.md:8]: @Otuyishime hi

**Comment by olaboutcode** [comment]
Nice places!!!

**Pull request merged**